lib/board.rb

```diff
@@ -0,0 +1,66 @@
+class Board
+  attr_accessor :cells, :display, :position, :turn_count, :update
+  def initialize
+    @cells = Array.new(9, " ")      #has an attribute to store the cells of the board
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.